SPL/SPL_c1.php

```diff
@@ -998,6 +999,7 @@ public function hasChildren(): bool {}
      * @return null|RecursiveIterator An SplFileObject does not have children so this method returns NULL.
      */
     #[TentativeType]
+    #[LanguageLevelTypeAware(['8.2' => 'null|null'], default: 'null|RecursiveIterator')]
```

Comment:
'8.2' => 'null|null' :) kek... As I understand this is due to unfinished implementation of PHP 8.2 and I hope it will be fixed but still..kek

Reply:
Yea, that's 🔥